modules/claims_api/lib/custom_error.rb

```diff
@@ -15,6 +16,12 @@ def build_error
       case @error
       when Faraday::ParsingError
         raise_backend_exception
+      when ::Common::Exceptions::GatewayTimeout,
+        Timeout::Error,
+        Faraday::TimeoutError,
+        Breakers::OutageException
```

Comment:
we'll still need to handle the Net::HTTPGatewayTimeout as mentioned in the ticket.